tests/unit_tests/test_share_recons.py

```diff
@@ -118,7 +123,7 @@ def test_recons_errorhandring(self):
     )
     def test_sharize_recons(self, secrets: list):
         """ nパーティのシェア化，復元が正しくできてるかTest"""
-        for party_size in range(2, 10):
+        for party_size in range(2, 3):
```

Comment:
Is this mistake?  Could you tell me why you narrowed down the scope of the test?

Reply:
This is a mistake. This is a remnant from the time of debugging.
Thanks for finding it! I fixed it!

Reply:
@mdonaka Thank you!  I checked it!